src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
-export interface Figure {
 
+type FigureShape = 'triangle' | 'circle' | 'rectangle';
+type FigureColor = 'red' | 'green' | 'blue';
+
+export interface Figure {
+  shape: FigureShape;
+  color: FigureColor;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: FigureShape = 'triangle';
```

Comment:
and afterwards, use it here, to avoid writing string literals. Also redundant public modifier, fields are public by default, fix it everywhere
```suggestion
  shape: FigureShape = FigureShape.Triangle;
```